README.md

```diff
@@ -38,12 +38,12 @@ pip install --extra-index-url https://download.pytorch.org/whl/test/cu118 llama-
 To install from source e.g. for development use this command. We're using hatchling as our build backend which requires an up-to-date pip as well as setuptools package.
 ```
 pip install -U pip setuptools
-pip install --extra-index-url https://download.pytorch.org/whl/test/cu118 -e .
+pip install --extra-index-url https://download.pytorch.org/whl/test/cu118 git+https://github.com/facebookresearch/llama-recipes
```

Comment:
@daniellepintz thanks for the PR, usually installing from src have been addressed by `-e .` to make it easier for any further changes, and usually has the assumption of user cloned the repo already, can you pls elaborate a bit on the change.

Reply:
Hi Hamid, sorry about that. Originally those commands weren't working for me but I just tried again in a fresh environment and now they are working. So I removed those changes and made this PR just about fixing some nits. Thanks!